.github/workflows/oxipng.yml

```diff
@@ -58,13 +63,16 @@ jobs:
           toolchain: ${{ matrix.toolchain }}
           override: true
           components: clippy, rustfmt
-      - name: Cache cargo registry
-        uses: actions/cache@v1
+      - name: Cache cargo
+        uses: actions/cache@v3
         with:
-          path: ~/.cargo/registry/cache
-          key: ${{ matrix.target }}-${{ matrix.toolchain }}-cargo-registry-${{ hashFiles('**/Cargo.lock') }}
-          restore-keys: |
-            ${{ matrix.target }}-${{ matrix.toolchain }}-cargo-registry-
+          path: |
```

Comment:
This is taken from https://github.com/actions/cache/blob/main/examples.md#rust---cargo

Let me know if you want me to make any changes :)